common/thorhelper/thorcommon.cpp

```diff
@@ -1276,6 +1279,31 @@ class CRowStreamReader : public CSimpleInterfaceOf<IExtRowStream>
             filterRow = new RtlDynRow(*actual);
         }
     }
+
+    //interface IVirtualFieldCallback
+    virtual const char * queryLogicalFilename(const void * row) override
+    {
+        if (fieldCallback)
+            return fieldCallback->queryLogicalFilename(row);
+        return ""; //MORE - should this value be passed in rather than relying on the callback?
```

Comment:
MORE - is that for a later Jira, addressed since the comment was first added, or what?